rakelib/default_plugins.rb

```diff
@@ -119,6 +119,8 @@ module RakeLib
       logstash-input-stdin
       logstash-input-tcp
       logstash-output-stdout
+      logstash-output-null
+      logstash-codec-rubydebug
```

Comment:
interesting, if I remove the `rubydebug`, funny thing `rake test:core` hangs, while bin/rspec not :disappointed: